lib/vquic/ngtcp2.c

```diff
@@ -395,7 +409,117 @@ static int quic_init_ssl(struct quicsocket *qs)
   gnutls_server_name_set(qs->ssl, GNUTLS_NAME_DNS, hostname, strlen(hostname));
   return 0;
 }
+#elif defined(USE_WOLFSSL)
+
+static void wolfssl_logging(const int logLevel, const char *const logMessage)
+{
+    fprintf(stderr, " ** wolfssl[%d]: %s\n", logLevel, logMessage);
+}
+
+static WOLFSSL_CTX *quic_ssl_ctx(struct Curl_easy *data)
+{
+  struct connectdata *conn = data->conn;
+  WOLFSSL_CTX *ssl_ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method());
+
+#if 0
```

Comment:
Is there any reason for leaving this disabled sequence in the code?

Reply:
See comment above.